docs/team/jordankanghm.md

```diff
@@ -5,24 +5,68 @@ title: "Jordan Kang's Project Portfolio Page"
 
 ### Project: iVolunteer
 
-iVolunteer is a...
+iVolunteer is an application for volunteer coordinators to manage volunteers, events, and their interactions. The user interacts with it using a CLI, with a GUI created with JavaFX. It is written in Java with about 14 kLoC.
 
 Given below are my contributions to the project.
 
-* **New Feature**: to be added soon
+* **New Feature**: Create Event Feature
+  * Provided the **foundational code** for creating volunteering events in our application. This is important as the main goal of our application is to allow volunteer coordinators to be able to store their events, along with the critical details associated with them. The following shows the aspects involved when implementing this feature.
+    * **Parsing of parameter arguments**, such as ensuring that the budget for an event is non-negative and in 2 decimal places to replicate real-life scenarios.
+    * **Creating new classes** to represent the fields of an event, namely `EventName`, `Role`, `DateTime` `Location`, `Material` and `Budget`.
+    * Implementing **checking for valid values** for the new classes created.
+    * Ensuring that the newly created `Event` can be **added to the Model seamlessly**.
+    * Ensuring that Event data can be **written into and retrieved from storage files** successfully.
+    * **Writing test cases** to ensure the feature works correctly.
 
-* **New Feature**: to be added soon
+  * As the original code we were provided with dealt with `Person` objects, I had to create and implement the logic for `Event` objects entirely as the main purpose of our app pertained to being able to **create volunteering events**. The experience was extremely **time-consuming** and took me a few days to implement as the existing code for `Person` was quite significant. However, it was also **informative** as it really allowed me to understand on a deeper level how the original code works and allowed me to implement the logic for `Event` more easily.
```

Comment:
`extremely **time-consuming** and took me a few days` indeed sir

Reply:
HAHAHAHAHA